src/backend/src/services/reimbursement-requests.services.ts

```diff
@@ -166,6 +167,16 @@ export default class ReimbursementRequestService {
       }
     });
 
+    /** Gets the current content of the .env file */
+    const currentEnv = require('dotenv').config().parsed;
+    const financeTeamId = currentEnv.FINANCE_TEAM_ID;
```

Comment:
you should get the env variable like this: `process.env.FINANCE_TEAM_ID`

Reply:
you should also check that it's not undefined/ throw an error if it is, I would move that code/ this code to get the finance team id inside the sendSlackReimbursementRequestNotification method you mande